plio/views.py

```diff
@@ -94,6 +122,15 @@ def unique_users(self, request, uuid):
             ]
         )
 
+    def is_user_in_org(self, organization_shortcode):
```

Comment:
this should not be inside plio/views.py - should be within some file in `users/`